parsec/logging.py

```diff
@@ -63,6 +66,8 @@ def dropper(logger, method_name, event_dict):
 
 
 def configure_sentry_logging(sentry_url):
-    sentry_handler = SentryHandler(sentry_url, level="WARNING")
-    root_logger = logging.getLogger()
-    root_logger.addHandler(sentry_handler)
+    sentry_logging = LoggingIntegration(
+        level=logging.WARNING,  # Capture info and above as breadcrumbs
```

Comment:
"capture warning and above as breadcrumbs" ?

Reply:
Fixé